core/frontend/src/components/vehiclesetup/configuration/accelerometer/FullAccelerometerCalibration.vue

```diff
@@ -0,0 +1,228 @@
+        Accelerometer Calibration
+      </v-card-title>
+      <div class="ma-6">
+        <p>
+          Full accelerometer calibration requires you to place the vehicle onto all of its sides and keep
+          it still for a few seconds.
+          This will allow the autopilot to calibrate the accelerometer.
+        </p>
+      </div>
+      <v-alert
+        v-if="![states.IDLE, states.WAITING_FOR_VEHICLE_RESPONSE].includes(state)"
+        dense
+        :type="current_state_type"
+      >
+        <strong>{{ current_state_text }}</strong>
+      </v-alert>
+      <v-card-actions class="justify-center">
+        <v-btn v-if="show_start_button" :loading="start_button_loading" class="primary" @click="startCalibration">
+          Start Calibration
+        </v-btn>
+        <v-btn v-if="show_next_button" class="primary" @click="nextStep">
+          Next
+        </v-btn>
+        <v-btn v-if="show_ok_button" class="primary" @click="cleanup">
+          Ok
+        </v-btn>
+      </v-card-actions>
+    </v-card>
+  </v-dialog>
+</template>
+
+<script lang="ts">
+
+import mavlink2rest from '@/libs/MAVLink2Rest'
+import { MavCmd } from '@/libs/MAVLink2Rest/mavlink2rest-ts/messages/mavlink2rest-enum'
+import autopilot_data from '@/store/autopilot'
+
+import { calibrator, PreflightCalibration } from '../../calibration'
+
+// Ideally these would come from mavlink2rest-enum
+// but we don't have the numbering information there
+enum AccelcalVehiclePos {
+  ACCELCAL_VEHICLE_POS_LEVEL = 1,
+  ACCELCAL_VEHICLE_POS_LEFT = 2,
+  ACCELCAL_VEHICLE_POS_RIGHT = 3,
+  ACCELCAL_VEHICLE_POS_NOSEDOWN = 4,
+  ACCELCAL_VEHICLE_POS_NOSEUP = 5,
+  ACCELCAL_VEHICLE_POS_BACK = 6,
+  ACCELCAL_VEHICLE_POS_SUCCESS = 16777215,
+  ACCELCAL_VEHICLE_POS_FAILED = 16777216
+}
+
+enum CalState {
+  IDLE,
+  WAITING_FOR_VEHICLE_RESPONSE,
+  WAITING_FOR_LEVEL_POSITION,
+  WAITING_FOR_LEFT_POSITION,
+  WAITING_FOR_RIGHT_POSITION,
+  WAITING_FOR_NOSEDOWN_POSITION,
+  WAITING_FOR_NOSEUP_POSITION,
+  WAITING_FOR_BACK_POSITION,
+  SUCCESS,
+  FAIL,
+}
+
+export default {
+  name: 'FullAccelerometerCalibration',
+  components: {
+  },
+  data() {
+    return {
+      dialog: false,
+      state: CalState.IDLE,
+    }
+  },
+  computed: {
+    states() {
+      return CalState
+    },
+    show_next_button() {
+      return ![
+        CalState.IDLE,
+        CalState.FAIL,
+        CalState.SUCCESS,
+        CalState.WAITING_FOR_VEHICLE_RESPONSE,
+      ].includes(this.state)
+    },
+    show_ok_button() {
+      return [CalState.SUCCESS, CalState.FAIL].includes(this.state)
+    },
+    show_start_button() {
+      return [CalState.IDLE, CalState.WAITING_FOR_VEHICLE_RESPONSE].includes(this.state)
+    },
+    start_button_loading() {
+      return this.state === CalState.WAITING_FOR_VEHICLE_RESPONSE
+    },
+    current_state_text() {
+      const AccelStateText: { [key: number]: string } = {
+        [CalState.WAITING_FOR_LEVEL_POSITION]: 'Place the vehicle on a level surface',
```

Comment:
The message isn't clear enough: we _must_ say what position we expect the vehicle to be. A _level_ surface can mean anything.